File: spec/code_samples/Php_http2/boards@hidden_entries/patch.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$body = new http\Message\Body;
$body->append('{"entry_id":["<ADD STRING VALUE>"]}');

$request->setRequestUrl('https://www.directfreight.com/api/boards/hidden_entries');
$request->setRequestMethod('PATCH');
$request->setBody($body);

$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE',
  'content-type' => 'application/json',
  'accept' => 'application/json'
));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: spec/code_samples/Python_python3/boards@hidden_entries/patch.py
import http.client

conn = http.client.HTTPSConnection("www.directfreight.com")

payload = "{\"entry_id\":[\"<ADD STRING VALUE>\"]}"

conn.request("PATCH", "/api/boards/hidden_entries", payload)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))